app/install.php

```diff
@@ -362,9 +362,24 @@ function printStep0() {
 <?php
 }
 
+function getProcessUsername() {
+	if (function_exists('posix_getpwuid') && function_exists('posix_geteuid')) {
+		$processUser = posix_getpwuid(posix_geteuid());
+		return $processUser['name'];
+	}
+
+	exec('whoami', $output);
```

Comment:
This function might be disabled, e.g. on shared hosting, like the previous ones, via https://php.net/ini.core#ini.disable-functions
Please check that this does not crash

Reply:
I've made a test with a dummy function and it does nothing.

Reply:
When `exec` is put in the list of disabled functions in `php.ini` ?

Reply:
I haven't thought about that! I'll look into it

Reply:
I have just tested and it solely produces a warning:

```
PHP Warning:  exec() has been disabled for security reasons in Standard input code on line 2
```

But you need to declare `$output` prior to using it

Reply:
I've added a test to see whether or not the function exist prior to call it